contents/javascript/Javascript-framework-Redux.md

```diff
@@ -0,0 +1,226 @@
+<frontmatter>
+  title: Redux
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+  
+{{ booktitle | safe }}
+
+# Introduction to Redux
+
+**Author: [Labayna Neil Brian Narido](https://github.com/nbriannl)** <br>
+Reviewers: 
+
+## What is Redux?
```

Comment:
Could we restructure this section to provide a better organization than the linked articles?
e.g.
- To understand what is redux, you first need to understand what problem it solves
- Give a anecdotal example of a complex situation which readers might have encountered when developing ( your own diagrams could be nice here )
- Introduce concept of application state with reference to the earlier example
- What does redux do that solves this?